core/templates/dev/head/header_css_libs.html

```diff
@@ -15,4 +15,7 @@
 {% endif %}
 <link rel="stylesheet" type="text/css" media="screen"
       href="/css/oppia.css">
+<link rel="stylesheet" type="text/css" media="screen"
```

Comment:
For now i am writing all the third_party_override.css in oppia.css and i am filing an issue for splitting css into context-specific files.